src/components/CreateTeam/form.js

```diff
@@ -13,18 +13,21 @@ const FormikEnhancer = withFormik({
   enableReinitialize: true,
 
   // validateOn
-  mapPropsToValues: () => ({
-    selectedClient: -1,
-    teamName: '',
-  }),
+  mapPropsToValues: (props) => {
+    const { initialFormValues } = props;
+    return ({
+      selectedClient: initialFormValues.selectedClient,
+      teamName: initialFormValues.teamName,
+    });
+  },
```

Comment:
I wouldn't even bother passing the initial values down as a prop here.

Reply:
As explained before, they are used to setup and update Formik.

Reply:
They have been removed now that resetForm is being passed up instead of prop values changing on createTeam success.